Makefile

```diff
@@ -1,4 +1,4 @@
-NAME=leap
+NAME=tumbleweed
```

Comment:
This doesn't look right.

Reply:
Good thing you noticed, mindless copying and pasting is not the best idea ;)